### DISCUSSION TITLE
[firebase_auth] Support disabling reCaptcha on Web

### DISCUSSION THREAD

**Discussion by itnks** [firebase/flutterfire#3789] [Feature request]
The previous ticket [[1942]](https://github.com/FirebaseExtended/flutterfire/issues/1942) that asked for this feature was closed. The last release [0.18.1+1] only Implemented this feature on iOS and not the web platform. It would be very nice if we could skip the reCaptcha step to improve the development experience.

**Comment by brampersandon** [comment]
++, for various reasons, the ReCaptcha implementation here is really onerous. Even as a dev, I can't pick all the fire hydrants, bicycles, or buses in frame! There has to be a way to tune this to reduce friction.
